frontend/src/organisms/BaseFeeChart.tsx

```diff
@@ -134,9 +143,12 @@ function LiveChart(props: BaseFeeChartProps) {
           <HStack><Text color="brand.secondaryText" fontWeight="bold">Rewards:</Text><BigNumberText number={block.rewards} /></HStack>
           {item.name === "issuance" && <HStack><Text color="brand.secondaryText" fontWeight="bold">Issuance:</Text><BigNumberText number={payload.issuance.abs()} /></HStack>}
           <HStack><Text color="brand.secondaryText" fontWeight="bold">Tips:</Text><BigNumberText number={block.tips} /></HStack>
-          <HStack><Text color="brand.secondaryText" fontWeight="bold">Basefee:</Text><BigNumberText number={block.baseFee} /></HStack>
+          <HStack><Text color="brand.secondaryText" fontWeight="bold">BaseFee:</Text><BigNumberText number={block.baseFee} /></HStack>
+          <HStack><Text color="brand.secondaryText" fontWeight="bold">PriorityFee:</Text><BigNumberText number={block.priorityFee} /></HStack>
           <HStack><Text color="brand.secondaryText" fontWeight="bold">Txs:</Text><Text>{block.transactions}</Text></HStack>
-          {item.name === "gas target" && <HStack><Text color="brand.secondaryText" fontWeight="bold">Gas Used:</Text><BigNumberText number={block.gasUsed} forced="wei" /></HStack>}
+          <HStack><Text color="brand.secondaryText" fontWeight="bold">GasTarget:</Text><GasTarget gasTarget={block.gasTarget} /></HStack>
+          <HStack><Text color="brand.secondaryText" fontWeight="bold">GasUsed:</Text><GasUsed gasUsed={block.gasUsed} /></HStack>
+          <HStack><Text color="brand.secondaryText" fontWeight="bold">GasUsedPct:</Text><GasUsedPercent gasUsed={block.gasUsed} gasTarget={block.gasTarget} /></HStack>
```

Comment:
This hover is getting a big big now, do you think we can condense this @stevengcook

Reply:
i wasn't sure what to do here - perhaps just gas used pct?

Reply:
yea I think so too